packages/ckeditor5-easy-image/src/easyimage.js

```diff
@@ -18,9 +18,12 @@ import CloudServicesUploadAdapter from './cloudservicesuploadadapter';
  *
  * This is a "glue" plugin which enables:
  *
+ * * {@link module:easy-image/cloudservicesuploadadapter~CloudServicesUploadAdapter}.
+ *
+ * This plugin requires plugin to be present in the editor configuration:
```

Comment:
WDYT about `@dependency` annotation that could be rendered somehow by our documentation tool?

```js
/**
 * ...
 * @dependency {module:image/image~Image}
 * @dependency {module:image/imageupload~ImageUpload}
 * ...
 */
```

I am pretty sure that the dependency list mentioned in the API docs will be missed. As for now, it's ok but the mechanism could be improved in the future.

Reply:
Extracted to https://github.com/ckeditor/ckeditor5/issues/8645.